ingest-app/src/cmr/ingest/services/nrt_subscription/subscriptions.clj

```diff
@@ -36,13 +37,29 @@
   "Get the subscriptions from the database. This function primarily exists so that
   it can be stubbed out for unit tests."
   ([context]
-   (mdb-search/find-concepts context {:latest true
-                                      :concept-type :subscription
-                                      :subscription-type "granule"}))
+   (let [response (mdb2/find-concepts context
+                                      {}
+                                      :subscription
+                                      {:raw? true
+                                       :http-options {:query-params
+                                                      {:latest true
+                                                       :concept-type :subscription
+                                                       :subscription-type "granule"}}})]
+     (if (= (:status response) 200)
+       (:body response)
+       nil)))
   ([context coll-concept-id]
-   (mdb-search/find-concepts context {:latest true
-                                      :concept-type :subscription
-                                      :collection-concept-id coll-concept-id})))
+   (let [response (mdb2/find-concepts context
+                                      {}
+                                      :subscription
+                                      {:raw? true
+                                       :http-options {:query-params
+                                                      {:collection-concept-id coll-concept-id
+                                                       :latest true
+                                                       :concept-type :subscription}}})]
+     (if (= (:status response) 200)
```

Comment:
I think you can just use a `when` here instead of `if` with an explicit `nil`. `when` evaluates to `nil` if its argument is `false`.